pkg/init/cmd/service/runc.go

```diff
@@ -52,8 +75,32 @@ func runcInit(rootPath string) int {
 		}
 		pidfile := filepath.Join(tmpdir, name)
 		cmd := exec.Command(runcBinary, "create", "--bundle", path, "--pid-file", pidfile, name)
-		cmd.Stdout = os.Stdout
-		cmd.Stderr = os.Stderr
+
+		// stream stdout and stderr to respective files
+		// ideally we want to use io.MultiWriter here, sending one stream to stdout/stderr, another to the files
+		// however, this hangs if we do, due to a runc bug, see https://github.com/opencontainers/runc/issues/1721#issuecomment-366315563
+		// once that is fixed, this can be cleaned up
+		stdoutFile := filepath.Join(logDir, serviceType+"."+name+".out.log")
+		stdout, err := os.OpenFile(stdoutFile, os.O_RDWR|os.O_CREATE, 0644)
```

Comment:
With the change to use `dumpFile` I think this can be `O_WRONLY` now, can't it? (same for stderr)

Reply:
Yes, it can actually. Worth doing.